src/python/ensembl/io/genomio/utils/archive_utils.py

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Utils to deal with archived files (gzip)."""
+"""Utils to deal with archive files."""
 
 __all__ = ["SUPPORTED_ARCHIVE_FORMATS", "extract_file"]
```

Comment:
```suggestion
__all__ = ["SUPPORTED_ARCHIVE_FORMATS", "extract_file", "open_gz_file"]
```

Reply:
This change has been addressed in another PR (https://github.com/Ensembl/ensembl-genomio/pull/172), thus why I haven't changed it.